js/src/components/account-select-control/index.scss

```diff
@@ -1,5 +0,0 @@
-.gla-account-select-control {
```

Comment:
Checked and confirmed that there is no more usage of `gla-account-select-control` CSS class in the code base.